library/src/morpheus-master/sass/modules/swapped-view/_base.scss

```diff
@@ -2,16 +2,18 @@
 	.#{$namespace}portalWrapper{
 		color: $swapped-view-text-color;
 		&.swapped-view{
-			.#{$namespace}topHeader{
+			.#{$namespace}topHeader, .#{$namespace}headerLogo {
 				background: $swapped-view-primary;
 			}
+/*
 			.#{$namespace}siteHierarchy{
 				background: $background-color-secondary;
 				color: $background-color;
 				a {
 					color: $background-color;
 				}
 			}
+*/
```

Comment:
Can this block just be removed rather than commenting it out?

Reply:
Sorry about that.  I was't sure what to do with it.  As is, it was was causing the **tool**'s header to change color as well, see SAK-41138 screenshots.  I thought that maybe it should be changed to

.#{$namespace}-site-nav

instead, to make the site tabs colors change, but that looked bad because the tab menu item colors didn't match correctly and I wasn't sure about making more / bigger changes.  Please let me know if it should just be removed or if it should be changed to .#{$namespace}-site-nav and fix the colors... although perhaps that can be tackled in a separate JIRA?

Reply:
Does this fix work for you locally, @austin48? What changes the $swapped-view-enabled variable to true for any of these CSS rules in modules/swapped-view/_base.scss to be applied? I'm not seeing this currently happening for topHeader in the browser.

Reply:
See https://jira.sakaiproject.org/browse/SAK-41138

you need to change it in /library/src/morpheus-master/sass/_defaults.scss

Reply:
Oh, I see. It has to be manually changed and built by the institution. I thought something in the application would enable this. Gotcha now. Thanks.